main.js

```diff
@@ -0,0 +1,21 @@
+const mobileMenu = document.querySelector('.mobile-container');
+const navMenu = document.querySelector('.nav-container');
+const menuBar = document.querySelector('.menu_bar');
+const menuClose = document.querySelector('.menu-close');
+const about = document.querySelectorAll('.mobile-navlink');
+menuBar.addEventListener('click', () => {
+  mobileMenu.style.display = 'flex';
+  navMenu.style.display = 'none';
+});
+
+menuClose.addEventListener('click', () => {
+  mobileMenu.style.display = 'none';
+  navMenu.style.display = 'flex';
+});
+
```

Comment:
- [ ] [OPTIONAL] You have done such great job implementing previous review requested changes :clap:. Unfortunately there is another problem with `x` button, since it doesn't show on mobile view and there is also a `margin-left` for the menu that shows the portfolio from behind which is not similar to the figma design :framed_picture:

> ![image](https://user-images.githubusercontent.com/60979926/174929108-f863c6b8-72dd-4005-aff6-1b3dbd39a61f.png)